djenga/currency/rounding.py

```diff
@@ -1,3 +1,17 @@
+"""
+This module provides functions for rounding numbers as currency.
+
+>>> currency_round_down(1.9999)
+Decimal('1.99')
+>>> currency_round_up(1.9911)
+Decimal('2.00')
+>>> currency_round_up4(1.9911)
+Decimal('1.9912')
+>>> currency_round_half_up(1.9951)
+Decimal('2.00')
+>>> currency_round_half_up4(1.995555)
```

Comment:
I didn't see this in the first PR, but is there a reason not to have `currency_round_half_up(value, digits)` ?  If you needed to pass `half_up4` as an argument, you could still do `partial(currency_round_half_up,digits=4)`

Python has some of the best native currying.  I think something like half_up4 is usually inferior to just providing the generic.  It's not a "maximum optimization" thing like estimating SQRT, right?